pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -84,17 +82,6 @@ func addConversionFuncs(scheme *runtime.Scheme) {
 	}
 }
 
-// The following two PodSpec conversions functions where copied from pkg/api/conversion.go
-// for the generated functions to work properly.
-// This should be fixed: https://github.com/kubernetes/kubernetes/issues/12977
-func Convert_api_PodSpec_To_v1_PodSpec(in *api.PodSpec, out *v1.PodSpec, s conversion.Scope) error {
```

Comment:
So I confirmed that this was basically compilation issue before - with the new generator it seems work good without it.